### PR TITLE
Create new branch for test

### PR DESCRIPTION
**Pull request by omsoo**
1. Contains the .html file
2. Contains the .css file
3. Contains the test.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
```
Comment (Deepakdanger) [inline index.html:7]: - [ ] **Required** Let's add `CSS file link` here to reflect CSS code in html file.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @omsoo, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up linters). Please add these two thing in `.gitignore`.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] For future suggestion, Please consider:-
- Commit should be in present tense.
- Commit should start with capital leters.For example ;- `Add linter files`
- [ ] Please commit more often.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omsoo** [comment]
All my commits have always been in present tense but would consider the capitalization as suggested.

The .md file was also captured in the .gitignore file and as advised I have also added the node_modules directory

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:7]: Addressed ✔

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 #Specific files to ignore
-*.md
\ No newline at end of file
+*.md
+test.md
+/node_modules
```
Comment (mateo951) [inline .gitignore:4]: Addressed 😉

**Review by mateo951 — APPROVED** [review]
### Hi @omsoo, 

Well done making all the necessary changes requested by your previous reviewer 🚀
Your project is complete!  There is nothing else to say other than... it's time to merge it 🤓 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omsoo** [comment]
@mateo951, Many thanks for the feedback

**Pull request merged**